lib/package.json

```diff
@@ -32,19 +32,23 @@
     "test": "jest"
   },
   "dependencies": {
-    "@ledgerhq/wallet-api-client": "1.1.0",
-    "axios": "^1.3.4",
-    "bignumber.js": "^9.1.2"
+    "axios": "^1.3.4"
   },
   "devDependencies": {
+    "@ledgerhq/wallet-api-client": "1.1.0",
     "@types/jest": "^29.5.2",
     "@types/node": "^12.12.21",
     "@typescript-eslint/eslint-plugin": "^5.53.0",
+    "bignumber.js": "^9.1.2",
```

Comment:
Do we need repeat those dependencies here?

Reply:
Yup, this is just so we have access to it in our own code base for testing and typings.